api/api.ts

```diff
@@ -89,6 +89,77 @@ app.post('/upload/:tenant', async (req: express.Request, res: express.Response)
     return
 })
 
+app.get('/download/:tenant/:idFile', async (req: express.Request, res: express.Response) => {
+    const driverHeader = req.headers['driver'] as string
+    if (!driverHeader) {
+        console.error(`Missing driver header at ${new Date()}`)
+        res.status(400).send('Missing driver header')
+        return
+    }
+
+    const authHeader = req.headers['authorization']
+    if (!authHeader) {
+        console.error(`Missing Authorization header at ${new Date()}`)
+        res.status(401).send('Missing Authorization header')
+        return
+    }
+
+    const token = authHeader?.split(' ')[1]
+    if (!token) {
+        console.error(`Request missing bearer token at ${new Date()}`)
+        res.status(401).send('Missing bearer token')
+        return
+    }
+
+    const authParams = new URLSearchParams({
```

Comment:
It seems to me that `URLSearchParams` here is a leaky abstraction

Reply:
Issue #52 opened to treat this point later